src/index.html

```diff
@@ -9,11 +9,46 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Product cards</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
+<body class="body">
+  <div class="goods-card" data-qa="card">
+    <div class="goods-img">
+      <img src="./images/imac.jpeg" alt="GOODS">
```

Comment:
imagine you use screenreader and you can know what's on the page only with alt attribute, will you recognize what `GOODS` mean on this page?
P.S. for alt text use only lowercase